packages/suite/src/actions/wallet/transactionActions.ts

```diff
@@ -163,8 +163,8 @@ export const fetchTransactions =
         });
 
         if (result && result.success) {
-            const updatedAccount = accountActions.update(account, result.payload)
-                .payload as Account;
+            const updateAction = accountActions.update(account, result.payload);
+            const updatedAccount = updateAction.payload as Account;
```

Comment:
I'm not sure if I understand. Because the `updatedAccount` was processed by the `update` function while `account` was not? Or could you elaborate, please?